request_a_govuk_domain/request/constants.py

```diff
@@ -0,0 +1,10 @@
+"""
+Constants to be used across modules
+"""
+
+# This is a map of notify templates to their template ids
```

Comment:
"This map links our email types to email templates defined in GOV.UK Notify and identified by their Notify UUID"

Reply:
Done